src/contrib/cluster/Akka.DistributedData/Serialization/ReplicatedDataSerializer.cs

```diff
@@ -166,6 +168,7 @@ public override string Manifest(object o)
                 case IPNCounterDictionary _: return PNCounterMapManifest;
                 case IPNCounterDictionaryDeltaOperation _: return PNCounterMapDeltaOperationManifest;
                 case IORMultiValueDictionary _: return ORMultiMapManifest;
+                case IORMultiValueDictionaryDeltaOperation _: return ORMultiMapDeltaOperationManifest;
```

Comment:
Don't we also need an equivalent `FromBinary` operation here?

Reply:
I think I missed that, let me check

Reply:
Done